### PR TITLE
Primary API

### PR DESCRIPTION
**Pull request by LingYi0612**
## What
 Database:
create 4 tables: items, requests, user, category

Make the endpoint of
seller:
-  GET /users/{user_id}
-  POST /items

Buyer:
- POST “/requests”
- GET "/users/my_requests/{username}"

Modified:
change all [image] in tsx file to [image_filename]

Wait to do

- [ ] deal the database of user ==> make a login page?
- [ ] other endpoints involved with user_id/user_name ---> which to use?
- [ ] move all function into another file, this one is really ugly though _(´ཀ`」 ∠)_

### CODE REVIEW

File: python/main.py [hunk 1]
```diff
@@ -1,89 +1,197 @@
...
+                category_id INTEGER,
+                user_id INTEGER,
+                image_filename STRING,
+                status STRING,
+                FOREIGN KEY(category_id) REFERENCES category(category_id)
+                FOREIGN KEY(user_id) REFERENCES user(user_id)
+          )""")
+
+# category
+c.execute("""CREATE TABLE IF NOT EXISTS category (
```
Comment (karuna) [inline python/main.py:48]: `items` is plural but `category` is singular

File: python/main.py [hunk 2]
```diff
@@ -1,89 +1,197 @@
...
+               category_id INTEGER PRIMARY KEY,
```
Comment (karuna) [inline python/main.py:49]: in `items` you put `id`, but in `category` you put `category_id`

Comment (karuna) [reply python/main.py:49]: please try to be consistent in naming

File: typescript/simple-mercari-web/src/components/ItemList/ItemList.tsx [hunk 3]
```diff
@@ -45,6 +47,49 @@ export const ItemList: React.FC<Prop> = (props) => {
     }
   }, [reload]);
 
+<<<<<<< Updated upstream
```
Comment (karuna) [inline typescript/simple-mercari-web/src/components/ItemList/ItemList.tsx:50]: merge conflict?

**Comment by LingYi0612** [comment]
make a new version based on @ku-bo-mi's code

**Pull request closed without merging**